### DISCUSSION TITLE
Smart Home with Watcher | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1859] [Q&A]
# Smart Home with Watcher | Seeed Studio Wiki

It focuses on the use of Watcher in the smart home.

https://wiki.seeedstudio.com/integrate_watcher_to_ha/

**Comment by eerke2021** [comment]
Hi, both my Watchers connect to the Sensecraft integration fine using your guide (I am glad it remembers the IP and port between tasks as I can never remember my HA IP) but I can’t set up any automations using the Alarm. I can set up automations using the Temperature etc. is there an extra bit of setup I need to do?
